### PR TITLE
Updating GMU CS faculty

### PR DESCRIPTION
**Pull request by nguyenthanhvuh**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv"). **IF YOU DO NOT DO THIS, YOUR COMMIT WILL BE SUMMARILY REJECTED.**

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[a-z].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.** **DO NOT MODIFY `csrankings-[0-9].csv`, WHICH ARE BEING DEPRECATED. USE `csrankings-[a-z].csv` INSTEAD.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

### CODE REVIEW

**Comment by nguyenthanhvuh** [comment]
List of CS faculty at GMU: https://cs.gmu.edu/directory/by-category/faculty/  and https://cs.gmu.edu/directory/by-category/affiliated-faculty/ (note: affiliated faculty can advise CS students)

File: csrankings-e.csv [hunk 1]
```diff
@@ -664,6 +664,7 @@ Evelyn Lulis,DePaul University,http://www.cdm.depaul.edu/about/Pages/People/facu
 Evert Haasdijk,VU Amsterdam,http://www.cs.vu.nl/ci/node/391,qGWvb-EAAAAJ
 Evgenia Smirni,College of William and Mary,http://www.cs.wm.edu/~esmirni,1Ed4HYUAAAAJ
 Evgenia Ternovska,Simon Fraser University,http://www.cs.sfu.ca/~ter,NOSCHOLARPAGE
+Evgenios Kornaropoulos,George Mason University,https://evgeniosk.github.io,2s0r_AkAAAAJ
```
Comment (emeryberger) [inline csrankings-e.csv]: Web page doesn't work.

File: csrankings-g.csv [hunk 2]
```diff
@@ -496,7 +496,7 @@ Giulio Jacucci,University of Helsinki,https://researchportal.helsinki.fi/en/pers
 Giulio Malavolta [SP],Max Planck Society,https://sites.google.com/view/giuliomalavolta,_pYEarUAAAAJ
 Giuseppe A. Di Lucca,University of Sannio,https://www.ding.unisannio.it/persone/docenti/di-lucca,pHMonFwAAAAJ
 Giuseppe Antonio Di Lucca,University of Sannio,https://www.ding.unisannio.it/persone/docenti/di-lucca,pHMonFwAAAAJ
-Giuseppe Ateniese,Stevens Institute of Technology,http://web.stevens.edu/facultyprofile/?id=2182,EyZJ08MAAAAJ
+Giuseppe Ateniese,George Mason University,https://ateniese.github.io,EyZJ08MAAAAJ
```
Comment (emeryberger) [inline csrankings-g.csv]: Web page still says Stevens. Please point to something indicating new affiliation.

File: csrankings-j.csv [hunk 3]
```diff
@@ -1589,6 +1589,7 @@ Jonathan Sun,University of Southern Mississippi,http://www.cs.usm.edu/~jsun,NOSC
 Jonathan Turner,Washington University in St. Louis,http://www.arl.wustl.edu/~jst,Cgx9W6UAAAAJ
 Jonathan Ventura,UCCS,http://jventura.net,k0E68OgAAAAJ
 Jonathan Walpole,Portland State University,http://web.cecs.pdx.edu/~walpole,-pS6P-oAAAAJ
+Jonathan Warrell,George Mason University,https://sites.google.com/site/jonathanwarrell,3KHjDQ4AAAAJ
```
Comment (emeryberger) [inline csrankings-j.csv]: No indication of GMU affiliation on web page.

File: csrankings-x.csv [hunk 4]
```diff
@@ -138,6 +138,7 @@ Xiaojun Wan 0001,Peking University,http://www.icst.pku.edu.cn/lcwm/wanxj,lTTeBdk
 Xiaojun Ye,Tsinghua University,http://www.thss.tsinghua.edu.cn/publish/soften/3131/2010/20101219111626359565665/20101219111626359565665_.html,NOSCHOLARPAGE
 Xiaojun Zhai,University of Essex,https://www.essex.ac.uk/people/zhaix77307/xiaojun-zhai,sLYyja0AAAAJ
 Xiaokang Yang,Shanghai Jiao Tong University,https://scholar.google.com/citations?user=yDEavdMAAAAJ=zh-CN,yDEavdMAAAAJ
+Xiaokuan Zhang,George Mason University,http://web.cse.ohio-state.edu/~zhang.5840/,HbuPL5IAAAAJ
```
Comment (emeryberger) [inline csrankings-x.csv]: No indication of GMU affiliation on home page.

File: csrankings-x.csv [hunk 5]
```diff
@@ -316,6 +317,7 @@ Xueping Peng,University of Technology Sydney,https://www.uts.edu.au/staff/xuepin
 Xueqi Cheng,Chinese Academy of Sciences,http://www.bigdatalab.ac.cn/~cxq,hY8aLqAAAAAJ
 Xueqin Liu,Queen's University Belfast,https://www.qub.ac.uk/schools/eeecs/Connect/Staff/BusinessCard/?name=xueqin.liux,bl6s9ZYAAAAJ
 Xueqing Liu 0001,Stevens Institute of Technology,https://www.cs.stevens.edu/~xliu127,g5W4khgAAAAJ
+Xuesu Xiao,George Mason University,https://www.cs.utexas.edu/~xiao/,bWbsbjAAAAAJ
```
Comment (emeryberger) [inline csrankings-x.csv]: Not joining until Fall 2022; please submit a PR in January.

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by nguyenthanhvuh** [comment]
@emeryberger thanks,  all mentioned problems should be fixed now.  Most of those who don't mention GMU on their websites are joining next year and so I just remove them in this PR as you said and will do another PR for them later.

**Review by emeryberger — APPROVED** [review]

**Pull request merged**